core/src/main/scala/org/apache/spark/rdd/DoubleRDDFunctions.scala

```diff
@@ -74,6 +74,22 @@ class DoubleRDDFunctions(self: RDD[Double]) extends Logging with Serializable {
   }
 
   /**
+   * Compute the population standard deviation of this RDD's elements.
+   * @since 2.1.0
```

Comment:
Sure. No doubt about that.
I'm just curious whether we need to replace them someday.
Anyway, I'm almost fixed.